src/components/ValidateCode/ValidateCodeModal.tsx

```diff
@@ -50,12 +50,13 @@ function ValidateCodeModal({code, accountID}: ValidateCodeModalProps) {
                         <Icon
                             width={variables.modalTopIconWidth}
                             height={variables.modalTopIconHeight}
-                            src={Illustrations.MagicCode}
+                            src={MagicCode}
                         />
                     </View>
                     <Text style={[styles.textHeadline, styles.textXXLarge, styles.textAlignCenter]}>{translate('validateCodeModal.title')}</Text>
                     <View style={[styles.mt2, styles.mb2]}>
                         <Text style={styles.textAlignCenter}>
+                            {translate('validateCodeModal.doNotShare')}
```

Comment:
<img width="976" alt="image" src="https://github.com/user-attachments/assets/65eef12d-05b6-436a-9f18-c1326a4990c3" />

Can you remove it please
```suggestion
```